src/pages/Home/components/AutocompleteInput/AutocompleteInput.tsx

```diff
@@ -30,16 +30,16 @@ const AutocompleteInput = ({ onChange, value }: Props) => {
       <Select
         options={data}
         isLoading={isLoading}
-        onInputChange={value => handleInputChange(value)}
+        onInputChange={(value) => handleInputChange(value)}
```

Comment:
**question** why this change was made?

Reply:
When I saved this file it happened automatically. I think it's a prettier

Reply:
**question** maybe it's better not to create a function  and replace it with `onInputChange={handleInputChange}`?